src/crud/useFetchMediaUrl.ts

```diff
@@ -125,21 +126,22 @@ export const useFetchMediaUrl = (reporter?: any) => {
           ) as MediaFile;
           if (mediarec && mediarec.attributes) {
             if (cancelled()) return;
+            var local = { localname: '' };
             const audioUrl =
               mediarec.attributes.audioUrl ??
               mediarec.attributes.s3file ??
               mediarec.attributes.originalFile;
-            const path = await dataPath(audioUrl, PathType.MEDIA);
-            const foundLocal = await ipc?.exists(path);
-            if (foundLocal || !accessToken) {
+            const path = await dataPath(audioUrl, PathType.MEDIA, local);
+            const foundLocal = local.localname === path;
+            if (foundLocal || offline) {
               if (!path.startsWith('http')) {
                 if (cancelled()) return;
                 dispatch({
                   payload: await safeURL(path),
                   type: MediaSt.FETCHED,
                 });
                 return;
-              } else if (!accessToken) {
+              } else if (offline) {
                 dispatch({
                   payload: 'no offline file',
```

Comment:
This should be a localized error

Reply:
It's localized by the receiver of the media_state

Reply:
if (mediaState.error.startsWith('no offline file'))
        showMessage(sharedStr.fileNotFound);